

<launch>
    <arg name="config"             default="configs/Legacy.yaml" />
    <arg name="jit"                default="True" />
    <arg name="input_image_topic"  default="/camera1/image_raw" />
    <arg name="result_topic"       default="/camera1/detected_lanes" />

    <node name="lane_detector" pkg="lane_detector" type="detect_node.py">
        <param name="config"             type="str"  value="$(arg config)" />
        <param name="jit"                type="bool" value="$(arg jit)" />
        <param name="input_image_topic"  type="str"  value="$(arg input_image_topic)" />
        <param name="result_topic"       type="str"  value="$(arg result_topic)" />
    </node>
</launch>
